wp-api.js

```diff
@@ -40,59 +35,49 @@
 		};
 	}
 
-	function WP_API_Utils() {
-		var origParse = Date.parse,
-			numericKeys = [ 1, 4, 5, 6, 7, 10, 11 ];
-
 
-		this.parseISO8601 = function( date ) {
-			var timestamp, struct, i, k,
-				minutesOffset = 0;
+	var origParse = Date.parse;
 
-			// ES5 §15.9.4.2 states that the string should attempt to be parsed as a Date Time String Format string
-			// before falling back to any implementation-specific date parsing, so that’s what we do, even if native
-			// implementations could be faster
-			//              1 YYYY                2 MM       3 DD           4 HH    5 mm       6 ss        7 msec        8 Z 9 ±    10 tzHH    11 tzmm
-			if ((struct = /^(\d{4}|[+\-]\d{6})(?:-(\d{2})(?:-(\d{2}))?)?(?:T(\d{2}):(\d{2})(?::(\d{2})(?:\.(\d{3}))?)?(?:(Z)|([+\-])(\d{2})(?::(\d{2}))?)?)?$/.exec(date))) {
-				// avoid NaN timestamps caused by “undefined” values being passed to Date.UTC
-				for ( i = 0; ( k = numericKeys[i] ); ++i) {
-					struct[k] = +struct[k] || 0;
-				}
+	/**
+	 * Parse date into ISO8601 format
+	 * 
+	 * @param {Date} date
+	 */
+	wp.api.utils.parseISO8601 = function( date ) {
+		var timestamp, struct, i, k,
+			minutesOffset = 0,
+			numericKeys = [ 1, 4, 5, 6, 7, 10, 11 ];;
+
+		// ES5 §15.9.4.2 states that the string should attempt to be parsed as a Date Time String Format string
+		// before falling back to any implementation-specific date parsing, so that’s what we do, even if native
+		// implementations could be faster
+		//              1 YYYY                2 MM       3 DD           4 HH    5 mm       6 ss        7 msec        8 Z 9 ±    10 tzHH    11 tzmm
+		if ((struct = /^(\d{4}|[+\-]\d{6})(?:-(\d{2})(?:-(\d{2}))?)?(?:T(\d{2}):(\d{2})(?::(\d{2})(?:\.(\d{3}))?)?(?:(Z)|([+\-])(\d{2})(?::(\d{2}))?)?)?$/.exec(date))) {
```

Comment:
We could technically [assemble this regex as a string](https://github.com/processing/p5.js/blob/dbef89fdaa6c94648c534b4a0e869b63611e63c9/src/color/p5.Color.js#L298-L308) for more readability: the way this is documented works pretty well, though, especially given that this is not code that is prone to change